packages/react-i18n/src/hooks/useTranslate.js

```diff
@@ -0,0 +1,4 @@
+import { useContext } from 'react';
+import { Context } from '../context/i18n.context';
+
+export const useTranslate = () => useContext(Context);
```

Comment:
Maybe we could `useMemo` to avoid multiple calls on translate function ?

Reply:
🤔 not sure, it looks like micro optimisation to me.
If it brinks perf issues then we will.